app/templates/admin/createEvent.html

```diff
@@ -20,20 +20,32 @@
 {% set isNewEvent = 'create' in request.path %}
 <!-- Program manager isCeltsAdmin and they are a manager for the program  -->
 <div class="text-center mb-3">
-    {% if isNewEvent %}
-      {% if template.tag == 'single-program' %}
-        {% set page_title = 'Create Event for ' + eventData["program"].programName %}
-      {% elif template.tag == 'all-volunteer' %}
-        {% set page_title = 'Create All Volunteer Training' %}
-      {% elif template.tag == 'no-program' %}
-        {% set page_title = 'Create Other CELTS-Sponsored Event' %}
-      {% else %}
-        {% set page_title = 'Create ' + template.name + ' Event' %}
-      {% endif %}
-      {% else %}
-        {% set page_title = eventData.name %}
-      {% endif %}
+{% if isNewEvent %}
+    {% if template.tag == 'single-program' %}
+      {% set page_title = 'Create Event for ' + eventData["program"].programName %}
+    {% elif template.tag == 'all-volunteer' %}
+      {% set page_title = 'Create All Volunteer Training' %}
+    {% elif template.tag == 'no-program' %}
+      {% set page_title = 'Create Other CELTS-Sponsored Event' %}
+    {% else %}
+      {% set page_title = 'Create ' + template.name + ' Event' %}
+    {% endif %}
+{% else %}
+     {% set programNameforEvent = eventData["program"].programName %}
+     {% if eventData.name == 'All Volunteer Training' and program == None %}
+        <h1>{{eventData.name}}</h1>
+     {% elif eventData.program != None %}
```

Comment:
This conditional is a double negative and could be changed to just `{% elif eventData.program %}'` if possible.

Reply:
fixed